src/customers/components/CustomerAddressListPage/CustomerAddressListPage.tsx

```diff
@@ -36,10 +36,9 @@ const useStyles = makeStyles(
       width: 600
     },
     root: {
-      columnGap: theme.spacing(3),
       display: "grid",
-      gridTemplateColumns: "repeat(3, 1fr)",
-      rowGap: theme.spacing(3)
+      gap: `${theme.spacing(3)}px`,
+      gridTemplateColumns: "repeat(3, 1fr)"
```

Comment:
Oh, it's not mentioned in task, but I see it looks terrible on mobile screen. May we also add `grid-template-columns: repeat(1, 1fr);` and/or `grid-template-columns: repeat(2, 1fr);` for smaller resolutions?